youtube_base/youtubers/templates/youtubers/home.html

```diff
@@ -1,9 +1,23 @@
-
 {% block title %}
 
 {% endblock %}
 {% block content %}
 <div class="container mt-4">
+    {% if user.is_authenticated %}
+        <div class="user-info">
+            Ви ввійшли як<a class="username" href="/profile"> {{ user.username }}</a>
+            <form action="{% url 'logout' %}" method="post">
+                {% csrf_token %}
+                <button type="submit" class="btn btn-danger">Вийти</button>
+            </form>
+        </div>
+    {% else %}
+    <p>Don't have an account? Sign up <a href="{% url 'sign_up' %}">here</a>!</p>
```

Comment:
Let's correct the indentation.

The if block HTML headers have indentation, but the else block does not have it.